techmatters/flex-plugins
# Move webchat into flex-plugins

Primary reviewer:

## Description
Move `webchat` repo into `flex-plugins`.

GitHub Action changes:
- Remove individual deploy actions ✅ 
  - This reduces unnecessary action spamming
  - Keep `aselo development` actions, since they're run more often
-  Unify build CI action
- Unify Release tags action


### Related Issues
Fixes https://tech-matters.atlassian.net/browse/CHI-1755

### Verification steps
- Check deploy actions still work fine
- Check Build CI action builds both projects
- What to do on release tag actions?
